src/staff-administrator/components/staff-add-edit-user/staff-add-edit-user-form/staff-add-edit-user-form.component.ts

```diff
@@ -314,6 +314,20 @@ export class StaffAddEditUserFormComponent implements OnInit {
     }
   }
 
+  public ngAfterViewInit(): void {
+    this.fragmentFocus();
```

Comment:
Is this needed? Can't we enable anchorScrolling in routerOptions?
https://angular.io/api/router/ExtraOptions

Reply:
Agree with @MateiHMCTS, @connorpgpmcelroy  Is it too much work?

Reply:
Agreed to have a tech debt for this

Reply:
Agreed